stratum/hal/lib/tdi/tdi_sde_table_entry.cc

```diff
@@ -65,10 +65,12 @@ ::util::Status TdiSdeWrapper::InsertTableEntry(
    */
 
   const auto flags = ::tdi::Flags(0);
-  RETURN_IF_TDI_ERROR(table->entryAdd(
+  tdi_status_t status = table->entryAdd(
       *real_session->tdi_session_, *dev_tgt, flags, *real_table_key->table_key_,
-      *real_table_data->table_data_))
-      << "Could not add table entry with: " << dump_args();
+      *real_table_data->table_data_);
+  if (!((status == BF_SUCCESS) || (status == BF_ALREADY_EXISTS)))
+     return ::util::Status(::util::error::Code::INTERNAL,
```

Comment:
Why do this instead of  `return MAKE_ERROR(code) << message`?

Reply:
Right, done